stable/jenkins/README.md

```diff
@@ -73,8 +73,8 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `Master.JMXPort`                  | Open a port, for JMX stats           | Not set                                                                      |
 | `Master.ExtraPorts`               | Open extra ports, for other uses     | Not set                                                                      |
 | `Master.OverwriteConfig`          | Replace config w/ ConfigMap on boot  | `false`                                                                      |
-| `Master.HostName`                 | Enables chart ingress, ingress host path      | Not set                                                                         |
-| `Master.Ingress.ApiVersion`      | Ingress api version                  | Not set                                                                         |
+| `Master.HostName`                 | Ingress host path      | Not set                                                                         |
+| `Master.Ingress.Enabled`          | Enables ingress      | `false`                                                                         |
```

Comment:
This is starting to look so much better!
Can we change the entire ingress section to follow the [naming best practices](https://helm.sh/docs/chart_best_practices/#naming-conventions)?

Reply:
lowercase and camelcase? Wouldn't that be completely inconsistent with the rest of this chart then?

Reply:
it will, but my goal is to have the entire chart follow the best practices. it is a work in progress :)

Reply:
So something like;

Reply:
So something like;
`Master.ingress.enabled`  
`Master.ingress.hostName` 
`Master.ingress.annotations` 
`Master.ingress.labels`    
`Master.ingress.path` 
`Master.ingress.TLS`

Not sure about TLS versus tls versus tLS :D

Reply:
Yes, just like that! TLS is the way to go :)

Reply:
let me correct myself, `tls` is the way to go :)

Reply:
done